web/src/ambient.d.ts

```diff
@@ -16,6 +16,11 @@ export type MovieType = {
   watched: boolean;
 };
 
+export type Movies = {
+  movies: MovieType[];
+  count: number;
```

Comment:
`count` is never sent to the frontend though I think, if it is it's most certainly a mistake by me